python/cudf/cudf/tests/test_binops.py

```diff
@@ -1110,11 +1125,19 @@ def test_datetime_dateoffset_binaryop(
     expect = op(psr, poffset)
     got = op(gsr, goffset)
 
+    if is_timezone_aware_dtype(dtype):
+        expect = expect.dt.tz_convert("UTC")
+        got = got.dt.tz_convert("UTC")
```

Comment:
Shouldn't we be asserting that both are the same time in `Asia/Kathmandu`? Why do we need to convert to UTC first?

Reply:
Ah OK this is a pytz vs zoneinfo thing.

You can keep the comparison in UTC, but could you add

```python
assert isinstance(expect.dtype, pd.DatetimeTZDtype)
assert str(expect.dtype.tz) == str(got.dtype.tz)
```

before the conversion so we can check they returned the "same" timezone?